src/ifcwrap-csharp/IfcOpenShell.Net/IfcOpenShell.TestApp/IfcOpenShellUtils.cs

```diff
@@ -0,0 +1,110 @@
+using System.Collections.Immutable;
```

Comment:
Have a look at this commented-out code for the entire file, please.

Consider adding a comment why this exists / how to use or removing the file / test app completely